app/controllers/categories_controller.rb

```diff
@@ -0,0 +1,58 @@
+class CategoriesController < ApplicationController
+
+  def index
+    @categories = Category.all
+  end
+
+  def show
+    @category = Category.find(params[:id])
+    @jobs = @category.jobs
+  end
+
+  def new
+    @category = Category.new
+    @jobs = Job.all
+  end
+
+  def create
+    @category = Category.new(category_params)
+    @jobs = Job.all
+    if @category.save
+      flash[:success] = "#{@category.title} added!"
+      redirect_to category_path(@category)
+    else
+      render :new
```

Comment:
Don't forget to handle the errors. What happens if a user enters something incorrectly and the Category doesn't get created? The user should see a nice error message.